core/scripts/sql/texera_ddl.sql

```diff
@@ -112,4 +115,34 @@ CREATE TABLE IF NOT EXISTS workflow_version
     `creation_time`      TIMESTAMP                   NOT NULL DEFAULT CURRENT_TIMESTAMP,
     PRIMARY KEY (`vid`),
     FOREIGN KEY (`wid`) REFERENCES `workflow` (`wid`) ON DELETE CASCADE
+) ENGINE = INNODB;
+
+CREATE TABLE IF NOT EXISTS project
+(
+    `pid`			INT UNSIGNED AUTO_INCREMENT 	NOT NULL,
```

Comment:
Can we make them aligned?

Reply:
Updated spacing to make project tables aligned